src/mono/wasm/templates/templates/browser/app-support.js

```diff
@@ -149,44 +147,43 @@ function applyArguments() {
     }
 }
 
-let toAbsoluteUrl = function(possiblyRelativeUrl) { return possiblyRelativeUrl; }
+let toAbsoluteUrl = function (path, prefix) {
+    if (prefix.startsWith("/")) {
+        return path;
+    }
+    return prefix + path;
+}
 const anchorTagForAbsoluteUrlConversions = document.createElement('a');
-toAbsoluteUrl = function toAbsoluteUrl(possiblyRelativeUrl) {
-   anchorTagForAbsoluteUrlConversions.href = possiblyRelativeUrl;
-   return anchorTagForAbsoluteUrlConversions.href;
+toAbsoluteUrl = function toAbsoluteUrl(path, prefix) {
+    anchorTagForAbsoluteUrlConversions.href = prefix + path;
+    return anchorTagForAbsoluteUrlConversions.href;
 }
 
-let loadDotnetPromise = import('/dotnet.js');
-let argsPromise = fetch('/runArgs.json')
-                    .then(async (response) => {
-                        if (!response.ok) {
-                            console.debug(`could not load /args.json: ${response.status}. Ignoring`);
-                        } else {
-                            runArgs = await response.json();
-                            console.debug(`runArgs: ${JSON.stringify(runArgs)}`);
-                        }
-                    })
-                    .catch(error => console.error(`Failed to load args: ${stringify_as_error_with_stack(error)}`));
-
-Promise.all([ argsPromise, loadDotnetPromise ]).then(async ([ _, { default: createDotnetRuntime } ]) => {
+try {
+    initRunArgs();
```

Comment:
We want to load to populate `runArgs` with contents from `runArgs.json`, then `initRunArgs`, and then rest of `applyArguments`, because we want to have `runArgs.json` as the default values, and fill in, or override as needed.